File: myonlinefm/src/main/java/com/sec/myonlinefm/classificationprogram/data/FavData.java
package com.sec.myonlinefm.classificationprogram.data;

import android.graphics.Bitmap;

/**
 * Created by SRC-TJ-MM-BinYang on 2018/5/10.
 */

public class FavData {

    private int channelID;
    private int categoryID;
    private String channelName;
    private Bitmap channelIcon;
    private String podCasterName;

    public int getCategoryID() {
        return categoryID;
    }

    public void setCategoryID(int categoryID) {
        this.categoryID = categoryID;
    }

    public String getPodCasterName() {
        return podCasterName;
    }

    public void setPodCasterName(String podCasterName) {
        this.podCasterName = podCasterName;
    }

    public Bitmap getChannelIcon() {
        return channelIcon;
    }

    public int getChannelID() {
        return channelID;
    }

    public String getChannelName() {
        return channelName;
    }

    public void setChannelIcon(Bitmap channelIcon) {
        this.channelIcon = channelIcon;
    }

    public void setChannelID(int channelID) {
        this.channelID = channelID;
    }

    public void setChannelName(String channelName) {
        this.channelName = channelName;
    }
}

File: myonlinefm/src/main/java/com/sec/myonlinefm/classificationprogram/data/FavData.kt
package com.sec.myonlinefm.classificationprogram.data

import android.graphics.Bitmap


/**
 * Created by SRC-TJ-MM-BinYang on 2018/5/10.
 */
class FavData {
    internal var channelID = 0
    internal var categoryID = 0
    internal var channelName: String? = null
    internal var channelIcon: Bitmap? = null
    internal var podCasterName: String? = null
    fun getCategoryID(): Int {
        return categoryID
    }

    fun setCategoryID(categoryID: Int) {
        this.categoryID = categoryID
    }

    fun getPodCasterName(): String? {
        return podCasterName
    }

    fun setPodCasterName(podCasterName: String?) {
        this.podCasterName = podCasterName
    }

    fun getChannelIcon(): Bitmap? {
        return channelIcon
    }

    fun getChannelID(): Int {
        return channelID
    }

    fun getChannelName(): String? {
        return channelName
    }

    fun setChannelIcon(channelIcon: Bitmap?) {
        this.channelIcon = channelIcon
    }

    fun setChannelID(channelID: Int) {
        this.channelID = channelID
    }

    fun setChannelName(channelName: String?) {
        this.channelName = channelName
    }
}